lib/cli/list-version.js

```diff
@@ -55,7 +55,7 @@ module.exports = async () => {
   process.stdout.write(
     `Framework Core: ${version}${installationModePostfix}\n` +
       `Plugin: ${dashboardPluginVersion}\n` +
-      `SDK: ${sdkVersion}\n` +
+      `Platform Client: ${platformClientVersion}\n` +
```

Comment:
What is your opinion of the name of this item? Should we change the name to `Platform Client` or keep the old `SDK` but just change the source of the value? I'm a bit torn as it's currently Platform Client but the name might change back to SDK in the future

Reply:
:+1: changed